jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/insights/QueryEditor.kt

```diff
@@ -143,6 +143,14 @@ class QueryEditor internal constructor(
         return unit.second
     }
 
+    fun setSelectedTimeUnit(unit: ChronoUnit) {
+        for (knownUnits in timeUnits) {
```

Comment:
nit: seeing a `for` loops is weird, any reason to do this over something more functional like `timeunits.filter { unit == it.second }.forEach{relativeTimeUnit.selectedItem = knownUnits}`?

Reply:
Can we just use an enum instead? And an EnumComboBoxModel?